src/platforms/windows/windowscryptosettings.cpp

```diff
@@ -82,7 +82,7 @@ bool CryptoSettings::getKey(uint8_t key[CRYPTO_SETTINGS_KEY_SIZE]) {
     return true;
   }
 
-  logger.log() << "Invalid key";
+  logger.debug() << "Invalid key";
```

Comment:
This seems like something a bit more serious than a `debug()` message. Probably worthy of a `warning()`